lib/autoupdate/agent/setup.go

```diff
@@ -204,11 +222,15 @@ func (ns *Namespace) Teardown(ctx context.Context) error {
 	if err := svc.Disable(ctx); err != nil {
 		return trace.Wrap(err, "failed to disable teleport-update systemd timer")
 	}
-	if err := os.Remove(ns.updaterServiceFile); err != nil && !errors.Is(err, fs.ErrNotExist) {
-		return trace.Wrap(err, "failed to remove teleport-update systemd service")
-	}
-	if err := os.Remove(ns.updaterTimerFile); err != nil && !errors.Is(err, fs.ErrNotExist) {
-		return trace.Wrap(err, "failed to remove teleport-update systemd timer")
+	for _, p := range []string{
+		ns.updaterServiceFile,
+		ns.updaterTimerFile,
+		ns.dropInFile,
+		ns.needrestartConfFile,
```

Comment:
should we ignore removal this config if it is not exist? because when we write template we ignore write if there are no directory